catalog/admin/module/catalog-admin-module-maplayers/src/main/webapp/lib/react-mount/spec.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import Mount from './'
+
+import { mount } from 'enzyme'
+
+describe('<Mount />', () => {
+  it('should fire on when mounted', (done) => {
+    mount(<Mount on={() => done()} />)
```

Comment:
If the other method gets called instead, the test will timeout causing a test failure.